Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectListViewController.swift

```diff
@@ -5,29 +5,37 @@
 //  Created by PSU2 on 6/28/17.
 //  Copyright © 2017 Jaca. All rights reserved.
 //
-
 import UIKit
 
 class ProjectListViewController: UIViewController {
-
+    
     var currentUser: UserModel = UserModel()
     var projectList: ProjectListModel = ProjectListModel()
     var username = ""
     var password = ""
     var instance = ""
     var endpointString = ""
     @IBOutlet weak var collectionView: UICollectionView!
```

Comment:
Move @IBOutlet declarations to the top to be consistent with LoginViewController.swift?